doc/users/prev_whats_new/whats_new_3.6.0.rst

```diff
@@ -69,6 +70,7 @@ Using the ``layout='compressed'`` layout reduces the space between the Axes,
 and adds the extra space to the outer margins:
 
 .. plot::
+    :alt: 4 identical 2 by 2 heatmaps, each cell a different color: purple, blue, yellow, green going clockwise from upper left corner the 4 heatmaps are laid out in a 2 by 2 grid with minimum white space between the heatmaps
```

Comment:
Seems like there should be a period after 'corner'.

Spell out Four at the beginning of a sentence.